docs/5g-on-helium/5g-poc.mdx

```diff
@@ -119,20 +117,131 @@ must:
 
+A 5G Hotspot will perform a speed test approximately every 12 hours with the results being averaged
+by the 5G PoC Oracle over the previous `6` tests. This moving average is reset following a speed
+test gap of `>12 hours` and a minimum of `2` speed test results are required to be eligible for
+Genesis Rewards.
+
+:::note Moving Average
+
+The "Moving Average", also known as a "Rolling Average" is the average of the 6 most recent speed
+test results for each download, upload, and latency.
+
+:::
+
+#### Example Calculation Tables
+
+Here are a set of tables to help visualize all of these components that make up the Speed Test
+result.
+
+**Successful Speed Test Series:**
+
+Here all Speed Test Metrics are passing, the 5G Hotspot is providing a high quality connection.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-success.png')} />
+
+**Missing A Single Speed Test:**
+
+This 5G Hotspot has not reported Speed Test data, this will be common for a newly connected 5G
+Hotspot. This 5G Hotspot _will_ be rewarded for all Reward Periods.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-one-miss.png')} />
+
+**Missing Multiple Speed Tests:**
+
+This 5G Hotspot has not reported a series of Speed Test data, this is often a result of a 5g Hotspot
+being disconnected from the internet for a long time. This 5G Hotspot _will not_ earn rewards during
+while Test 5 and 6 are the most recent.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-many-miss.png')} />
+
+**Failing A Speed Test:**
+
+This 5G Hotspot has "failed" its Speed Test on all metrics (orange cells), but the Moving Average
+(light green cells) stays within the passing range. This 5G Hotspot _will_ be rewarded for all
+Reward Periods.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-one-fail.png')} />
+
+**Failing Multiple Speed Tests:**
+
+This 5G Hotspot has failed many Speed Tests in a row (orange cells), this has caused the Moving
+Average to fail (light green cells). This 5G Hotspot _will not_ earn rewards during while Test 6, 7,
+and 8 are the most recent.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-many-fail.png')} />
+
+#### Helium Explorer 5G Hotspot Statistics Tab
+
+Current Speed Test Statistics for a given 5G Hotspot can be viewed on the Helium Explorer.
+
+<div class="container" style={{ padding: 0, paddingTop: 0 }}>
+  <div class="row">
+    <div class="col col--6" style={{ marginBottom: 0 }}>
+      <b>🟢 Speed Test Passing</b>
+      <p>
+        Here all Speed Test Metrics are passing, the 5G Hotspot is providing a high quality
+        connection.
+      </p>
+      <img
+        className="explorerSidebar"
+        src={useBaseUrl('img/5g-on-helium/explorer-5g-stats-pass.png')}
+      />
+    </div>
+    <div class="col col--6" style={{ marginBottom: 0 }}>
+      <b>🟣 Not Enough Data</b>
```

Comment:
Make bubble yellow to correspond to color coding in the tables, doesn't have to be exact color match.